src/modules/SearchPage/components/Pagination/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { usePaginationRange } from 'hooks';
+import { DOTS } from 'appConstants';
+import { MouseEvent } from 'react';
+import * as S from './styled';
+
+type Props = {
+  currentPage: number;
+  totalPageCount: number;
+  nextPage: () => void;
+  previousPage: () => void;
+  specificPage: (event: MouseEvent<HTMLButtonElement>) => void;
+};
+export const Pagination = (props: Props) => {
+  const { totalPageCount, currentPage, nextPage, previousPage, specificPage } = props;
+  const paginationRange = usePaginationRange({
+    totalPageCount,
+    currentPage,
+  });
```

Comment:
Next time return objects from your hooks.
```suggestion
  const {paginationRange = []} = usePaginationRange({
    totalPageCount,
    currentPage,
  });
```

Reply:
done